### DISCUSSION TITLE
upload plugin

### DISCUSSION THREAD

**Discussion by mincheol-jeong** [Budibase/budibase#9303] [Help] [answered]
Hi, I am currently using Budibase https://github.com/chungchunwang/Budibase-File-Upload plugin, but I keep getting payload too large error. Even though mysql converts to longtext and max_allowed_packet is set to 1GB, an error occurs when a 4MB file is inserted. Is there a way?

**Comment by melohagan** [comment, marked as answer]
Hey @mincheol-jeong 

I think this would be best raised as an issue on the plugin repo!
